search-api/tests/postman/search-api.postman_collection.json

```diff
@@ -2125,6 +2125,164 @@
 					]
 				}
 			]
+		},
+		{
+			"name": "internal",
+			"item": [
+				{
+					"name": "Add/Update business in solr",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = pm.response.json()\r",
+									"\r",
+									"pm.test(\"Status code is 201/Created\", function () {\r",
+									"    pm.response.to.have.status(201);\r",
+									"    \r",
+									"    // success, so set keys\r",
+									"    pm.environment.set(\"access_request_id\", jsonData.id)\r",
+									"    pm.environment.set(\"document_key\", jsonData.documents[0].documentKey)\r",
```

Comment:
probably don't want to set any envs in this call?

Reply:
ya. Removed it